jenkins-x.yml

```diff
@@ -19,43 +19,51 @@ pipelineConfig:
                 dir: /workspace/source/env
                 command: jx
                 args: ['step','verify','preinstall']
-              - name: jx-crds
+              - name: install-jx-crds
                 command: jx
                 args: ['upgrade','crd']
-              - name: kube-system
+              - name: install-nginx
                 dir: /workspace/source/systems/jxing
                 command: jx
                 args: ['step','helm','apply', '--boot', '--remote', '--no-vault', '--name', 'jxing']
                 env:
                   - name: DEPLOY_NAMESPACE
                     value: kube-system
-              - name: helm-cluster-values
+              - name: create-install-values
                 dir: /workspace/source/env
                 command: jx
                 args: ['step','create','install', 'values', '-b']
-              - name: install-vault
-                dir: /workspace/source/systems/vault
+              - name: install-external-dns
```

Comment:
@cagiti This step needs to be conditionally executed only when `externalDNS` is enabled. The BDD tests are currently failing when the externalDNS is enabled. I am looking to find a solution for this.

Reply:
yeah, would be good to perhaps wrap the contents of the files that step executes with a condition so if you haven't got it enables it basically no-ops

Reply:
Any idea how is the best to do this. The values is defined in `jx-requirements.yml` and I don't see how the values are propagated into the `jenkins-x.yml` pipeline file. 

It seems that some commands are loading the `jx-requirements.yml` file and are parsing a specific key from it. e.g `step boot vault` is looking for `secretStore: vault`.